Packs/ReplaceMatchGroup/Scripts/ReplaceMatchGroup/ReplaceMatchGroup_test.py

```diff
@@ -0,0 +1,32 @@
+import demistomock as demisto
+import pytest
+
+
+test_data = [
+    ('[{"Sha1Hash"},{"437"},{"Sha1Hash"}]', '\{".*?"(\},\{)".*?"\}', '@@@', '[{"Sha1Hash"@@@"437"},{"Sha1Hash"}]'),
+    ('{"1"},{"2"},{"3"},{"4"}', '\{".*?"\}(,)\{".*?"\}', ':', '{"1"}:{"2"},{"3"}:{"4"}'),
+    ('this is a test, this is another test.', '.*?(,).*?(\.)', '!', 'this is a test! this is another test!'),
+    ('a, b. c,.', '.*?(,|\.)', '!', 'a! b! c!!'),
+    ('.this is a test, this is another test.', '(\.).*?(,).*?(\.)', '!', '!this is a test! this is another test!'),
+    ('[{"UrlValue:https://sslmanageamazones.ddns.net/info.ppl.service.access/paypal"}, '
+     '{"UrlValue:https://sslmanageamazones.ddns.net/info.ppl.service.access/paypal"}, '
+     '{"UrlValue:https://sslmanageamazones.ddns.net/info.ppl.service.access/paypal"}]', '.*?(:).*?:\\/\\/', '":"',
+     '[{"UrlValue":"https://sslmanageamazones.ddns.net/info.ppl.service.access/paypal"}, '
+     '{"UrlValue":"https://sslmanageamazones.ddns.net/info.ppl.service.access/paypal"}, '
+     '{"UrlValue":"https://sslmanageamazones.ddns.net/info.ppl.service.access/paypal"}]')
```

Comment:
add a test case in which no matches are found and verify string is returned as the value given